app/views/school/before_can_order/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%- title = t('page_titles.school_before_can_order') %>
+<%- content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+
+    <%= render 'shared/chromebook_information_intro' %>
+    <%= render partial: 'shared/chromebook_information_form',
+      locals: {
+        url: school_chromebooks_path,
+        show_i_dont_know_option: true,
+        scope: 'page_titles.school_user_welcome_wizard.chromebooks',
+        form_object: @chromebook_information_form } %>
```

Comment:
after submission user is sent to `http://localhost:3000/school/details`. Is this the desired behaviour?

Reply:
No, they should go to the homepage

Reply:
I've made it go to the school homepage (`/school`)